qrcodereaderview/src/main/java/com/google/zxing/client/android/camera/CameraManager.java

```diff
@@ -137,20 +142,20 @@ public synchronized void setManualCameraId(int cameraId) {
   }
 
   /**
-   * @param newSetting if {@code true}, light should be turned on if currently off. And vice versa.
+   * @param enabled if {@code true}, light should be turned on if currently off. And vice versa.
    */
-  public synchronized void setTorch(boolean newSetting) {
+  public synchronized void setTorchEnabled(boolean enabled) {
     OpenCamera theCamera = openCamera;
     if (theCamera != null) {
-      if (newSetting != configManager.getTorchState(theCamera.getCamera())) {
+      if (enabled != configManager.getTorchState(theCamera.getCamera())) {
         boolean wasAutoFocusManager = autoFocusManager != null;
         if (wasAutoFocusManager) {
           autoFocusManager.stop();
           autoFocusManager = null;
         }
-        configManager.setTorch(theCamera.getCamera(), newSetting);
+        configManager.setTorchEnabled(theCamera.getCamera(), enabled);
         if (wasAutoFocusManager) {
-          autoFocusManager = new AutoFocusManager(context, theCamera.getCamera());
+          autoFocusManager = new AutoFocusManager(theCamera.getCamera());
```

Comment:
This is how original Zxing project is doing it. It seems like you need to create a new AutoFocusManager every time you change the configuration.